app/src/main/java/com/android/bookswap/data/source/network/BookFirestoreSource.kt

```diff
@@ -79,18 +101,36 @@ class BooksFirestoreSource(private val db: FirebaseFirestore) : BooksRepository
           callback(result)
         }
   }
-
+  /**
+   * Updates an existing book in the Firestore database.
+   *
+   * @param dataBook The DataBook object containing the updated book details.
+   * @param callback Callback to be invoked with the result of the operation. The result is Unit on
+   *   success, or an exception on failure.
+   */
   override fun updateBook(dataBook: DataBook, callback: (Result<Unit>) -> Unit) {
     performFirestoreOperation(
         db.collection(collectionBooks).document(dataBook.uuid.toString()).set(dataBook), callback)
   }
-
+  /**
+   * Deletes a book from the Firestore database.
+   *
+   * @param uuid The UUID of the book to be deleted.
+   * @param dataBook The DataBook object containing the book details.
+   * @param callback Callback to be invoked with the result of the operation. The result is Unit on
+   *   success, or an exception on failure.
+   */
   override fun deleteBooks(uuid: UUID, dataBook: DataBook, callback: (Result<Unit>) -> Unit) {
```

Comment:
This method deletes a single book but the name `deleteBooks` suggests that it would delete multiple books.
Additionally, the uuid parameter seems useless and could be removed during a future code cleanup